util/testplanner/class_defs.py

```diff
@@ -322,9 +324,12 @@ def results_table(self, regr_results, map_full_testplan=True, fmt="pipe"):
                     milestone, entry_name, test["name"], test["passing"],
                     test["total"], pass_rate
                 ])
+                if entry == self.entries[-1]:
+                    self.final_total = [
+                        self.name.upper(), test["passing"], test["total"],
+                        pass_rate
+                    ]
```

Comment:
This should be outside of both for loops. 

I think rather than assume that last entry is the "grand total", its better to do this instead:
```python
    if entry.milestone == "N.A." and entry.name == "N.A.":
        # This is the grand total
        self.final_total = ...
```
This would be inside the for loop on 313.

Reply:
that's right, didn't see 2 loop there.
Updated, thanks.